src/vs/workbench/browser/actions/listCommands.ts

```diff
@@ -600,7 +601,7 @@ KeybindingsRegistry.registerCommandAndKeybindingRule({
 KeybindingsRegistry.registerCommandAndKeybindingRule({
 	id: 'list.selectAll',
 	weight: KeybindingWeight.WorkbenchContrib,
-	when: ContextKeyExpr.and(WorkbenchListFocusContextKey, WorkbenchListSupportsMultiSelectContextKey),
+	when: ContextKeyExpr.and(WorkbenchListFocusContextKey, WorkbenchListSupportsMultiSelectContextKey, WEBVIEW_FOCUSED.negate()),
```

Comment:
Instead of bringing the webview concept all the way down here, why not simply use the `weight` concept to give priority to whatever command webview owns to be run when `Cmd A` gets hit?

Reply:
Good point. We could have a more complex/restrict when clause with a higher weight to override the list view command.

Reply:
Oh yes, that makes so much sense, didn-'t knot that was possible, thanks